package.json

```diff
@@ -87,6 +87,7 @@
     "shelljs": "~0.8",
     "source-map-explorer": "^2.5.2",
     "ts-loader": "^9.4.2",
+    "ts-node": "^10.9.1",
```

Comment:
very minor but since we already depend on esbuild i would prefer `esbuild-runner` over `ts-node`, it's waaay faster since it doesn't compile, just strips types

Reply:
Sure, have changed it. Still want to type-check the file though so have added a `tsc --noEmit` first.